std/basics.sy

```diff
@@ -0,0 +1,42 @@
+from common use (
```

Comment:
Rename the automatically imported one to `prelude` - basics and common are to similar in meaning. I don't easily know which one is the auto-imported one. 

Maybe add a comment about only use-statements being allowed in the prelude?